packages/manager/src/features/CloudPulse/Alerts/CreateAlert/NotificationChannels/AddChannelListing.tsx

```diff
@@ -140,16 +141,19 @@ export const AddChannelListing = (props: AddChannelListingProps) => {
   return (
     <Controller
       render={({ fieldState, formState }) => (
-        <>
-          <Typography marginBottom={1} marginTop={3} variant="h2">
+        <Stack mt={3} spacing={2}>
+          <Typography marginBottom={1} variant="h2">
             4. Notification Channels
           </Typography>
-          {(formState.isSubmitted || fieldState.isTouched) && fieldState.error && (
-            <Notice spacingBottom={0} spacingTop={12} variant="error">
-              {fieldState.error.message}
-            </Notice>
-          )}
-          <Stack spacing={1}>
+          {(formState.isSubmitted || fieldState.isTouched) &&
+            fieldState.error &&
+            fieldState.error.message && (
```

Comment:
```suggestion
            fieldState.error.message?.length && (
```

so it is handled for length as well and notice becomes free